files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml

```diff
@@ -1,105 +1,76 @@
 ---
-# ALL tags must be with dashes (-) instead of underscores (_)
+# NOTE: Use dashes (-) exclusively for tags and underscores (_) exclusively for destinations.
```

Comment:
This note is great. I would also move such rules in the readme of shared-tpv repo.

Reply:
Well destinations are not a thing on the shared-tpv repo, only scheduling tags, and the [README already says](https://github.com/galaxyproject/tpv-shared-database/blob/2c2e8596ad35c74b961a03df0d00e37a0b27e841/README.md?plain=1#L15).

> Dashes are used as separators in tags instead of underscores.

So I guess this is done already.